src/world.ts

```diff
@@ -39,7 +39,54 @@ export class World {
   private furniture: Array<ICollider> = [];
 
   constructor() {
-    this.init();
+    const w = document.getElementById("world");
```

Comment:
This is just moved up from the `init()` function for smoother coverage in the content.